Products/Brass-Beta/gWASM.md

```diff
@@ -51,7 +54,289 @@ In this quick tutorial you will
 
 [Follow installation instructions](/Products/Brass-Beta/Installation.md). Remember that it is required to run in the background during gWASM computations.
 
-#### 2. Create a subnet of nodes
+---
+
+#### 2. Prerequisites
+
+Make sure that you have **Rustlang** and **emscripten** installed on your machine. 
+
+* [Rust](https://www.rust-lang.org/tools/install) - rustup toolchain add nightly `rustup target add wasm32-unknown-emscripten --toolchain nightly` TODO [JK - Rust issue link]
```

Comment:
```suggestion
* [Rust](https://www.rust-lang.org/tools/install) - rustup toolchain add nightly `rustup target add wasm32-unknown-emscripten --toolchain nightly` (**NB** it is currently necessary to rely on the `nightly` toolchain due to a bug with the Emscripten target in Rust, which has already been fixed -- see [rust-lang/rust#67976](https://github.com/rust-lang/rust/pull/67976) for reference -- however, won't land in stable for a couple of releases. Once it lands in stable, we'll update the docs. Until then, please use the `nightly` toolchain to build your gWasm apps using Rust.)
```

Reply:
@nieznanysprawiciel is testing all the steps on Windows. Yesterday it worked on MacOs so it should also work smoothly on Linux systems.